src/components/AppToolbar/AddColumnButton.css

```diff
@@ -0,0 +1,22 @@
+.add-list-btn {
+    color: white !important;
```

Comment:
Too many `!important` declarations, maybe other types of style overriding will be better?

Reply:
Inline styles? In my opinion, this isn't a good idea. Is mix of js and css in one file better?